pyilc/input.py

```diff
@@ -127,6 +124,14 @@ def __init__(self, input_file=None):
             self.GN_FWHM_arcmin = np.asarray(p['GN_FWHM_arcmin'])
             assert len(self.GN_FWHM_arcmin) == self.N_scales - 1, "GN_FWHM_arcmin"
             assert all(FWHM_val > 0. for FWHM_val in self.GN_FWHM_arcmin), "GN_FWHM_arcmin"
+        elif self.wavelet_type == 'CosineNeedlets':  #Fiona added CosineNeedlets
+            # ellpeak values defining the gaussian needlets
```

Comment:
do you mean 'defining the cosine needlets'?

Reply:
Yes, sorry

Reply:
fixed this in later commit